src/components/InputItem/InputItem.module.css

```diff
@@ -1,3 +1,7 @@
 .wrap {
   padding-bottom: 15px;
 }
+
+.button {
+    margin-top: 10px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5715931)